app/models/manageiq/providers/azure/network_manager/floating_ip.rb

```diff
@@ -1,2 +1,5 @@
 class ManageIQ::Providers::Azure::NetworkManager::FloatingIp < ::FloatingIp
+  def self.available
```

Comment:
@djberg96 - I had wondered the same thing, but no:
https://github.com/ManageIQ/manageiq/blob/master/app/models/floating_ip.rb